migrace/copy_data.py

```diff
@@ -278,18 +281,20 @@ def reset_sequences(destination_host, destination_db, destination_user, destinat
         ("id", "externi_odkaz"),
         ("id", "samostatny_nalez"),
         ("id", "uzivatel_spoluprace"),
+        ("id", "tvar"),
     )
     for item in tables:
         destination_cursor.execute(f"SELECT SETVAL("
                                    f"(SELECT PG_GET_SERIAL_SEQUENCE('\"{item[1]}\"', '{item[0]}')),"
                                    f"(SELECT (MAX(\"{item[0]}\") + 1) FROM \"{item[1]}\"),FALSE);")
     destination_conn.commit()
+    print("Sequences are set.")
```

Comment:
@pesikj Je třeba ošetřit i sekvence pro: `osoba_ident_cely_seq`, `auth_user_ident_cely_seq`, `organizace_ident_cely_seq` a `heslar_ident_cely_seq`. To se zde myslím neřeší a bez toho nám řady ident_cely nebudou fungovat.